src/main/java/the/bytecode/club/bytecodeviewer/decompilers/bytecode/InstructionPrinter.java

```diff
@@ -245,8 +245,8 @@ protected String printJumpInsnNode(JumpInsnNode jin) {
                 + resolveLabel(jin.label);
     }
 
-    protected String printLineNumberNode() {
-        return "";
+    protected String printLineNumberNode(LineNumberNode lnn) {
+        return "// line " + lnn.line;
```

Comment:
Maybe we should add a setting for this?
Does not necessarily need to be in this PR, but we should make this customizable at some point at least :)